save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/basic/FiltersRow.kt

```diff
@@ -6,26 +6,39 @@ import org.cqfn.save.domain.TestResultStatus
 import org.cqfn.save.frontend.externals.fontawesome.faFilter
 import org.cqfn.save.frontend.externals.fontawesome.fontAwesomeIcon
 
+import org.w3c.dom.HTMLInputElement
 import org.w3c.dom.HTMLSelectElement
 import react.Props
+import react.dom.button
+import react.dom.defaultValue
 import react.dom.div
+import react.dom.input
 import react.dom.option
 import react.dom.select
 import react.fc
 
+import kotlinx.html.InputType
 import kotlinx.html.js.onChangeFunction
+import kotlinx.html.js.onClickFunction
 
 /**
- * A row of filter selectors for table with `TestExecutionDto`s. Currently the only filter is "status".
+ * A row of filter selectors for table with `TestExecutionDto`s. Currently filters are "status" and "test suite".
  *
- * @param initialValue initial value of `TestResultStatus`
+ * @param initialValueStatus initial value of `TestResultStatus`
+ * @param initialValueTestSuite initial value of `test suite`
  * @param onChange handler for selected value change
+ * @param onChangeTestSuite handler for input value
  * @return a function component
  */
+@Suppress("TOO_LONG_FUNCTION")
 fun testExecutionFiltersRow(
-    initialValue: String,
+    initialValueStatus: String,
+    initialValueTestSuite: String,
     onChange: (String) -> Unit,
```

Comment:
Come on, why rename `initialValue` and skip `onChange`?

Reply:
Yes. Forgot it :)